src/content/blog/astro-570.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: 'Astro 5.7'
+description: 'Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.'
```

Comment:
```suggestion
description: 'Astro 5.7 has a basket load of treats, including stable Sessions and SVG components and a new Experimental Fonts API.'
```

Reply:
I'm pretty sure it's fine to keep as a compound word